### ISSUE TITLE
`content.javascript.clipboard` doesn't work on a fresh start

### ISSUE THREAD

**Issue by rien333** [qutebrowser/qutebrowser#7281]
**Version info**:

```
qutebrowser v2.5.1
Git commit: 285a6f9a5 on makepkg (2022-06-21 21:46:17 +0200)
Backend: QtWebEngine 6.3.1, based on Chromium 94.0.4606.126 (from api)
Qt: 6.3.1

CPython: 3.10.5
PyQt: 6.3.1


```

**Does the bug happen if you start with `--temp-basedir`?**:

Yes.

**Description**

In my `config.py`, I have `content.javascript.clipboard` set to `access`.  However, this, by itself doesn't allow me to copy code blocks on github (such as the version info on top of this issue). To do so, I need to run `:set content.javascript.clipboard access/access-paste`  _after_ launching qutebrowser. 

In other words, it seems that the setting defaults to `none` at launch or something, despite the user's prefered default. `

**How to reproduce**

1. `/usr/bin/qutebrowser -T -s content.javascript.clipboard 'access'`
2. Try copying a code block from github (shouldn't work)
3. Run `:set content.javascript.clipboard access`
4. Now, copying a code block should work

**Comment by The-Compiler** [comment]
Looks like you got two for the price of one: `content.default_encoding` also doesn't work, and probably never did for four years (since 3956f81e730463adcba05d92d0043155609aa422) :sweat_smile:
